test/k8sT/manifests/external_service.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   labels:
     zgroup: external
     test: external
+  namespace: default
 spec:
```

Comment:
Won't this create a clusterIP? Is this the intention?

Reply:
I think it will not, but it's leftover from my debugging. Will remove